### PR TITLE
app icon for android & ios

### PR DESCRIPTION
**Pull request by hpereira98**
- adaptive icon foreground and background
- backup icon for android and ios if adaptive fails
- specific files created by the command:  _flutter packages pub run flutter_launcher_icons:main_

### CODE REVIEW

**Comment by BrunoMartins11** [comment]
Looks fine. When this PR is approved only merge it after PR #8 is merged.

**Review by BrunoMartins11 — DISMISSED** [review]

**Review by herulume — DISMISSED** [review]

**Review by BrunoMartins11 — APPROVED** [review]

**Review by herulume — APPROVED** [review]

**Pull request merged**
